docs/02_cleos/03_command-reference/wallet/import.md

```diff
@@ -1,55 +1,36 @@
 ## Command
+
+```sh
 cleos wallet import [OPTIONS]
+```
 
-**Where**
-* [OPTIONS] = See Options in Command Usage section below. 
+**Where**:
 
-**Note**: The arguments and options enclosed in square brackets are optional.
+* [`OPTIONS`] = See **Options** in **Command Usage** section below
 
 ## Description
-Imports private key into wallet. This command will launch `keosd` if it is not already running. 
+Use this command to imports private key into wallet. 
```

Comment:
imports => import

Reply:
Use this command to imports private key into wallet.  to
=>
Use this command to import a  private key into a (local maybe?) wallet.